Lib/test/test_clinic.py

```diff
@@ -1692,5 +1692,167 @@ def test_permute_optional_groups(self):
                 self.assertEqual(actual, expected)
 
 
+class FormatHelperTests(unittest.TestCase):
+
+    def test_strip_leading_and_trailing_blank_lines(self):
+        dataset = (
+            # Input lines, expected output.
+            ("a\nb",            "a\nb"),
+            ("a\nb\n",          "a\nb"),
+            ("a\nb ",           "a\nb"),
+            ("\na\nb\n\n",      "a\nb"),
+            ("\n\na\nb\n\n",    "a\nb"),
+            ("\n\na\n\nb\n\n",  "a\n\nb"),
+            # Note, leading whitespace is preserved:
+            (" a\nb",               " a\nb"),
+            (" a\nb ",              " a\nb"),
+            (" \n \n a\nb \n \n ",  " a\nb"),
+        )
+        for lines, expected in dataset:
+            with self.subTest(lines=lines, expected=expected):
+                out = clinic.strip_leading_and_trailing_blank_lines(lines)
+                self.assertEqual(out, expected)
+
+    def test_normalize_snippet(self):
+        snippet = """
+            one
+            two
+            three
+        """
+        template = (
+            "{indent}one\n"
+            "{indent}two\n"
+            "{indent}three"
+        )
+        for indent in 0, 4, 8:
+            expected = template.format(indent=" " * indent)
+            with self.subTest(snippet=snippet, indent=indent, expected=expected):
+                out = clinic.normalize_snippet(snippet, indent=indent)
+                self.assertEqual(out, expected)
+
+    def test_accumulator(self):
+        acc = clinic.text_accumulator()
+        self.assertEqual(acc.output(), "")
+        acc.append("a")
+        self.assertEqual(acc.output(), "a")
+        self.assertEqual(acc.output(), "")
+        acc.append("b")
+        self.assertEqual(acc.output(), "b")
+        self.assertEqual(acc.output(), "")
+        acc.append("c")
+        acc.append("d")
+        self.assertEqual(acc.output(), "cd")
+        self.assertEqual(acc.output(), "")
+
+    def test_c_repr(self):
+        dataset = "", "a", "abc"
+        for line in dataset:
+            expected = f'"{line}"'
+            with self.subTest(line=line, expected=expected):
+                out = clinic.c_repr(line)
+                self.assertEqual(out, expected)
+
```

Comment:
_This_ feels like testing for the sake of testing. The function seems too trivial to need dedicated unit tests imo; we can get test coverage for those lines through integration tests